pkg/apis/ocs/v1alpha1/storagecluster_types.go

```diff
@@ -25,7 +25,7 @@ type StorageDeviceSet struct {
 	Resources       corev1.ResourceRequirements  `json:"resources"`
 	Placement       rookalpha.Placement          `json:"placement"`
 	Config          StorageDeviceSetConfig       `json:"config,omitempty"`
-	DataPVCTemplate corev1.PersistentVolumeClaim `json:"volumeClaimTemplates"`
+	DataPVCTemplate corev1.PersistentVolumeClaim `json:"dataPVCTemplate"`
```

Comment:
To be consistent with Ceph, i'd expect this to be called `volumeClaimTemplate`.   This naming convention is also common within k8s... see the StatefulSet's volumeClaimTemplates list.